csharp/src/Drivers/BigQuery/BigQueryConnection.cs

```diff
@@ -434,7 +538,7 @@ private StructArray GetDbSchemas(
                 nullBitmapBuffer.Build());
         }
 
-        private StructArray GetTableSchemas(
+        StructArray GetTableSchemas(
```

Comment:
We should have a documented "preferred coding conventions". Much of the other C# code in this overall repo is explicitly using "private", and getting rid of it here has added a bunch of unrelated churn to this PR.